src/models/stores/ui.ts

```diff
@@ -32,6 +32,7 @@ export const UIModel = types
     activeGroupId: "",
     selectedTileIds: types.array(types.string),
     selectedCommentId: types.maybe(types.string),
+    scrollToTileId: types.maybe(types.string),
```

Comment:
I'd recommend making this an object that has both a tile id and a document identifier. For user docs this would be the key, for curriculum docs I think this would be its path. I think we have utilities to handle this. 

The addition of the document identifier would help with two things:
- possibility of curriculum docs with conflicting tile ids
- more efficient during the look up of tiles, if the scrollTo document identifier doesn't match the current document identifier the reaction can bail out immediately instead of iterating through all of its rows.